SofaKernel/modules/SofaBaseVisual/VisualModelImpl.cpp

```diff
@@ -529,11 +529,13 @@ bool VisualModelImpl::load(const std::string& filename, const std::string& loade
                 {
                     //Modified: previously, the texture coordinates were not loaded correctly if no texture name was specified.
                     //setMesh(*objLoader,tex);
+                    msg_warning() << "Loading obj mesh file directly inside the VisualModel will be deprecated soon. Use a MeshObjLoader and link the Data to the VisualModel.";
                     setMesh(*objLoader, true); 
                 }
                 else
-                {                    
-                    msg_error() << "Method setMesh is not anymore supported in sofa release 18.06.";
+                {
+                    msg_error() << "Loading mesh file directly inside the VisualModel is not anymore supported since release 18.06. Use a MeshLoader and link the Data to the VisualModel.";
+                    return false;
```

Comment:
I think we should put this error message at line 523, also.

Reply:
So why is obj still supported ?

Reply:
because I hadn't time to finish the work. :(
It was much more complicated than the other formats